samples/kotlin_compose/src/main/java/io/customer/android/sample/kotlin_compose/data/models/Configuration.kt

```diff
@@ -0,0 +1,20 @@
+package io.customer.android.sample.kotlin_compose.data.models
+
+import io.customer.sdk.CustomerIOConfig.Companion.AnalyticsConstants.AUTO_TRACK_DEVICE_ATTRIBUTES
+import io.customer.sdk.CustomerIOConfig.Companion.AnalyticsConstants.BACKGROUND_QUEUE_MIN_NUMBER_OF_TASKS
+import io.customer.sdk.CustomerIOConfig.Companion.AnalyticsConstants.BACKGROUND_QUEUE_SECONDS_DELAY
+import io.customer.sdk.CustomerIOConfig.Companion.AnalyticsConstants.SHOULD_AUTO_RECORD_SCREEN_VIEWS
+
+data class Configuration(
+    var siteId: String,
+    var apiKey: String
+) {
+    var trackUrl: String? = null
+
+    var backgroundQueueSecondsDelay: Double = BACKGROUND_QUEUE_SECONDS_DELAY
+    var backgroundQueueMinNumTasks: Int = BACKGROUND_QUEUE_MIN_NUMBER_OF_TASKS
+
+    var trackScreen: Boolean = SHOULD_AUTO_RECORD_SCREEN_VIEWS
+    var trackDeviceAttributes: Boolean = AUTO_TRACK_DEVICE_ATTRIBUTES
+    var debugMode: Boolean = false
```

Comment:
```suggestion
    var trackUrl: String? = null

    var backgroundQueueSecondsDelay: Double? 
    var backgroundQueueMinNumTasks: Int? 

    var trackScreen: Boolean?
    var trackDeviceAttributes: Boolean?
    var debugMode: Boolean?
```

I am assuming that the code `var backgroundQueueSecondsDelay: Double = BACKGROUND_QUEUE_SECONDS_DELAY` is giving a default CIO SDK config value in the app? Which means that the sample app is providing a default value that overrides the default value that is provided by the CIO SDK? 

If this assumption is true, I suggest that the sample app does not provide it's own set of default values. The sample app either configures the CIO SDK with a value that has been set in the settings screen (so the user of the app decided to override the SDK configuration) or the CIO SDK default value is used. 

From this suggestion, I would expect all of the `Configuration` values in this data class to be optional unless the user goes into the settings screen and changes something.

Reply:
>I am assuming that the code var backgroundQueueSecondsDelay: Double = BACKGROUND_QUEUE_SECONDS_DELAY is giving a default CIO SDK config value in the app? Which means that the sample app is providing a default value that overrides the default value that is provided by the CIO SDK?

We are referencing the same default values that the CIO SDK is using, hence we are not overriding it but using the same values. if the CIO SDK defaults change so would the app's default. And once user change the apps, app would then used the persisted ones.